docs/representing_statistics.md

```diff
@@ -125,3 +128,137 @@ The `StatisticalVariable` also makes consuming Data Commons data
+
+Sometimes, we want to make statements not about
+particular entities but about sets of entities of a particular type that share
+some properties, such as:
+
+1.  In 2016, there were 99999 people in USA, who were male,
+    married, with a median age of 22.
+2.  In 2017, there were 999 deaths in Travis County where the cause of death was
+    XYZ.
+
+The clauses "number of people who are male, hispanic" and "number of deaths
+where cause of death was XYZ", etc. are enumerations of variables about a
+specific population. The clauses "In 2016, there were 99999" and "In 2017, there
+were 999" are observations on those populations.
+
+In Data Commons, we define `StatisticalPopulation`s and `Observation`s.
+
+#### Representing `StatisticalPopulation`s
+
+A `StatisticalPopulation` is a set of instances of a certain type that
+satisfy some set of constraints. The property `populationType` is used to
+specify the type. Any property that can be used on instances of that type can
+appear on the `StatisticalPopulation`. An instance of `StatisticalPopulation` whose
+`populationType` is C1, which has the properties p1, p2, … with values v1, v2, …
+corresponds to the set of objects of type C1 that have the property p1 with
+value v1, property p2 with value v2, etc.
+
+For the two examples above, the MCF node
+
+```
+Node: dcid:StatisticalPopulationExample1
+typeOf: schema:StatisticalPopulation
+populationType: schema:Person
+location: dcid:country/USA
+gender: schema:Male
+maritalStatus: dcs:Married
+```
+encodes the clause "people in USA, who were male, married",
+and the MCF node
+
+```
+Node: dcid:StatisticalPopulationExample2
+typeOf: schema:StatisticalPopulation
+populationType: dcs:MortalityEvent
+location: dcid:geoId/48453
+causeOfDeath: XYZ
+```
+encodes the clause "deaths in Travis County where the cause of death was
+XYZ".
+
+StatisticalPopulationExample1(&2) is an abstract set--it does
+not correspond to a particular set of people who satisfy that constraint at a
+certain point in time, but rather, to an abstract specification, about which we
+can make observations that are grounded at a particular point in time. We now
+turn our attention to the representation of these observations.
+
+#### Representing `Observation`s
+
+Instances of the class `Observation` are used to specify observations about an
+entity (which may or may not be an instance of a `StatisticalPopulation`), at a
+particular time. The principal properties of an `Observation` are
+
+-   `observedNode`: the entity the data point applies to
+-   `measuredProperty`: what the observation is about
+-   `measuredValue`: the value of the observation
+-   `observationDate`: the date of, or last day of the observation
+-   `observationPeriod`: the length of time the observation took place
+
+For the same two examples, the MCF nodes
+
+```
+Node: dcid:Obs1
+type: schema:Observation
+observedNode: l:SP1
```

Comment:
How about referencing the above StatPop's actual localId? Here and elsewhere.

Reply:
Thanks, didn't notice this!